code/analyze/ac/stats/de_StatsFreqPreferences.m

```diff
@@ -79,9 +84,61 @@
         stats.(prop).std{si}(mi,:,:) = std(resps_by_prop, [], 3);
       end;
 
-      % Now, do the analysis as done in the `recfields` scripts.
-
-
     end;
-  end;
+    % Now, do the analysis as done in the `recfields` scripts.
+    
+    %Take raw stds and average over all units (see nn_2layer_processor)
+    raw_std = stats.freq.std{si};
+    %First, combine first 2 dimensions (models x hidden units)
+    raw_std = reshape(raw_std, [size(raw_std, 1)*size(raw_std, 2), size(raw_std, 3)]);
+    sm = mean(raw_std,1); %avg over each unit, like in recfields
+    ss  = std(raw_std,[],1)/sqrt(size(raw_std,1)); %stick to naming convention
+
+    stats.('freq').std_mean(si, :) = sm;
+    stats.('freq').std_std(si, :) = ss;
+    
+  end
+    
+  % To aid in readability and stay consistent w/ conventions in
+  % sigma_vs_crossover, define a few variables.
+  std_mean = stats.('freq').std_mean; % aids in readability, stay consistent w/ sigma_vs_crossover
+  numSigmas = n_sigmas;
+  cpi = dset.freqs;
+  stats.freq.cpi = cpi; %will need this later
+  
+  for ii=1:numSigmas
+    for ij=(ii+1):numSigmas
+      % which direction should we be searching for cross-overs?
+      ratios = std_mean(ii,:) ./ std_mean(ij,:); % Check when ratio goes over 1
```

Comment:
Which lines are you referring to, exactly? These lines are necessary to build `stats.('freq').xover` and `stats.('freq').spairs`, which are used in `de_PlotFreqPreferences`